libs/commonArrayFunctions.php

```diff
@@ -0,0 +1,19 @@
+<?php
```

Comment:
Usually it is better to move common functions like this into Helper classes with static methods